### PR TITLE
feat: upgraded Appwrite SDK to version 9

### PR DESCRIPTION
**Pull request by omeraplak**
Please provide enough information so that others can review your pull request:
The issue first started on our [Discord](https://discord.com/channels/837692625737613362/906206669787234334/1004270530997784596) server. Multiple database feature was added to Appwrite and it could not be used with `@pankod/refine-appwrite` package. 

### Test plan (required)

Demonstrate the code is solid. If not, please add `WIP:` in its title.

### Self Check before Merge

Please check all items below before review.

-   [ ] Corresponding issues are created/updated or not needed
-   [x] Docs are updated/provided or not needed
-   [x] Examples are updated/provided or not needed
-   [x] TypeScript definitions are updated/provided or not needed
-   [x] Tests are updated/provided or not needed
-   [x] Changesets are provided or not needed

### CODE REVIEW

File: .changeset/fresh-timers-rest.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+"@pankod/refine-appwrite": major
+---
+
+- Added `databaseId` support. The Default value is `default` for backward compability
+- Upgraded Appwrite SDK to 9
```
Comment (aliemir) [inline .changeset/fresh-timers-rest.md:6]:
```suggestion
- Upgraded Appwrite SDK to version 9
```

File: .changeset/fresh-timers-rest.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
```
Comment (aliemir) [inline .changeset/fresh-timers-rest.md:7]:
```suggestion

## Usage
```

File: .changeset/fresh-timers-rest.md (hunk 1 above)
Comment (aliemir) [reply .changeset/fresh-timers-rest.md:6]: Not really needed but reads better I think

File: .changeset/fresh-timers-rest.md (hunk 2 above)
Comment (aliemir) [reply .changeset/fresh-timers-rest.md:7]: Might look better in the changeling

**Review by aliemir — APPROVED** [review]

File: .changeset/fresh-timers-rest.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+"@pankod/refine-appwrite": major
+---
+
+- Added `databaseId` support. The Default value is `default` for backward compability
+- Upgraded Appwrite SDK to version 9
+
+## Usage
+
+```tsx
+import { Refine, AuthProvider } from "@pankod/refine-core";
```
Comment (omeraplak) [inline .changeset/fresh-timers-rest.md:11]:
```suggestion
import { Refine } from "@pankod/refine-core";
```

**Pull request merged**